config.sh

```diff
@@ -132,7 +132,7 @@ function run_tests {
         else
             python3 -m pip install numpy==1.20.3
         fi
-    else
+    elif [[ "$MB_PYTHON_VERSION" != 3.10 ]] || [[ "$PLAT" != "x86_64" ]]; then
         python3 -m pip install numpy
```

Comment:
Shall we use `--only-binary` so we don't need to remove the condition later?

Something like:
```sh
python3 -m pip install numpy --only-binary :all: || true
```

Reply:
We could, but it would also exclude "pypy3.6-7.3.3 focal i686", which does manage to successfully build numpy at the moment - https://github.com/python-pillow/pillow-wheels/runs/3332927842#step:4:6678

Reply:
OK, I thought it might have been added to save time. Sounds good, thanks!